.env.gold

```
@@ -1,5 +1,23 @@
-NEXT_PUBLIC_PROTOCOL_NAME="GLUON GOLD"
+NEXT_PUBLIC_FAVICON=/logo/gluon.png
 NEXT_PUBLIC_NEUTRON_TICKER=GAU
-NEXT_PUBLIC_PROTON_TICKER=GAUC
 NEXT_PUBLIC_PEG_ASSET=Gold
 NEXT_PUBLIC_PEG_UNIT=1g
+NEXT_PUBLIC_PROTON_TICKER=GAUC
+NEXT_PUBLIC_PROTOCOL_NAME="GLUON GOLD"
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_PROTOCOL_NAME key should go before the NEXT_PUBLIC_PROTON_TICKER key

(UnorderedKey)